docs/reference/predicates.md

```diff
@@ -128,6 +128,30 @@ Host(/^my-host-header\.example\.org$/)
 Host(/header\.example\.org$/)
 ```
 
+## Weight (priority)
+
+By default, the weight (priority) of a route is determined by the number of defined predicates.
+
+If you want to give a route more priority, you can give it more weight.
+
+Parameters:
+
+* Weight (int)
+
+Example where `route2` has more priority because it has more predicates:
+
+```
+route1: Path("/test") -> "http://www.zalando.de";
+route2: Path("/test") && PathRegexp(".*") -> "http://www.zalando.de";
+```
+
+Example where `route1` has more priority because it has more weight:
+
+```
+route1: Path("/test") && Weight(100) -> "http://www.zalando.de";
+route2: Path("/test") && PathRegexp(".*") && PathRegexp(".*") -> "http://www.zalando.de";
```

Comment:
Please replace PathRegexp() with True().
Can you also show an example where Weight() does not apply, for example a better Path/PathSubtree match?

Reply:
> Please replace PathRegexp() with True().

Done

> Can you also show an example where Weight() does not apply, for example a better Path/PathSubtree match?

Is this needed? I think this is clear enough?

Reply:
IMHO, it's fine like this. @szuecs is there a specific example on your mind?

Reply:
I thought like this:

```
r1: PathSubtree("/foo") && Weight(100) -> status(200) -> "<shunt>"
r2: PathSubtree("/foo/bar") -> status(201) -> "<shunt>"
```
This request will match r2: `GET /foo/bar HTTP/1.1`